src/app/pages/CpsSty/index.jsx

```diff
@@ -95,6 +97,14 @@ const CpsStyContainer = ({ pageData }) => {
     }
   `;
 
+  const StyledByline = styled(Byline)`
+    padding-bottom: ${GEL_SPACING_DBL};
+
+    @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+      padding-bottom: ${GEL_SPACING_TRPL};
+    }
+  `;
```

Comment:
Once again pretty much stolen from the timestamp: https://github.com/bbc/simorgh/pull/5512/files#diff-fdf1e6ecf0538604611b254a4f30cfd4R92-R98 

Similar to https://github.com/bbc/simorgh/pull/5512/files#r378138719 is there any guidance on how to choose these styles?

Another question, how do we choose what layer to apply styles? I've applied them in the __container__ to decide how they span the width of the page: https://github.com/bbc/simorgh/pull/5512/files#diff-157097b54aac0bc0bee88b1c4218c69bR18 and applied them in the __page__ here to decide how much space we have between elements on the page.

Reply:
My preference is to do as little in the container as possible, so I agree with what you've done here, because the container required additional markup and your styles there were related. The same for the page.